integration/src/binance/tx02.mainnet.signed.json

```diff
@@ -1,40 +1,36 @@
 {
-  "chain_id": "Binance-Chain-Tigris",
+  "account_number": "471113",
+  "chain_id": "Binance-Chain-Nile",
   "data": null,
   "memo": "",
   "msgs": [
     {
       "inputs": [
         {
-          "address": {
-            "type": "Buffer",
-            "data": [234, 93, 122, 233, 154, 156, 226, 249, 219, 124, 67, 169, 7, 55, 112, 36, 183, 243, 67, 232]
-          },
+          "address": "bnb1afwh46v6nn30nkmugw5swdmsyjmlxslgjfugre",
           "coins": [
             {
-              "denom": "BNB",
-              "amount": "1000"
+              "amount": "0.00001",
+              "denom": "BNB"
```

Comment:
The changes in field order and data type bring the test vectors in line with the `signBytes` value -- the serialized data that's actually signed. (Not to be confused with the serialized transaction, which is different for reasons.)